engines/mtropolis/elements.cpp

```diff
@@ -1553,12 +1553,14 @@ MiniscriptInstructionOutcome MToonElement::scriptSetCel(MiniscriptThread *thread
 }
 
 MiniscriptInstructionOutcome MToonElement::scriptSetRange(MiniscriptThread *thread, const DynamicValue &value) {
-	if (value.getType() != DynamicValueTypes::kIntegerRange) {
-		thread->error("Invalid type for mToon range");
-		return kMiniscriptInstructionOutcomeFailed;
-	}
+	if (value.getType() == DynamicValueTypes::kIntegerRange)
+		return scriptSetRangeTyped(thread, value.getIntRange());
+	if (value.getType() == DynamicValueTypes::kPoint)
+		return scriptSetRangeTyped(thread, value.getPoint());
 
 	return scriptSetRangeTyped(thread, value.getIntRange());
```

Comment:
Need to remove this line so it errors out if it's not an acceptable type instead of attempting to load the wrong type from the dynamic value.

Reply:
Done. Oops, this was a badly-resolved merge conflict that somehow still compiled :facepalm: